### DISCUSSION TITLE
[Feature] Reflect resources between kubernetes clusters

### DISCUSSION THREAD

**Discussion by amahdhaoui-ionos** [emberstack/kubernetes-reflector#480] [Ideas]
Reflecting resources between kubernetes clusters is a pretty common thing to do when working with a sharded k8s cluster setup. 

Your controller seems promising and I was wondering if cross-cluster reflection is on your roadmap?

**Comment by zchef2k** [comment]
I second this request. I recognize there are other projects that manage secrets across clusters, namely [external secrets](https://github.com/external-secrets/external-secrets) and the [secrets CSI driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver). These are good solutions, especially for environments that are widely distributed. However there is elegance and simplicity in this project's approach to cross-namespace secrets replication that hopefully can be extended to include remote clusters. A straightforward option to replicate secrets, or any object for that matter, from one to many clusters ought to exist that does not depend on additional external resources.
